open_seq2seq/utils/utils.py

```diff
@@ -381,7 +427,8 @@ def check_params(config, required_dict, optional_dict):
         vals = string_types
       if vals and isinstance(vals, list) and config[pm] not in vals:
         raise ValueError("{} has to be one of {}".format(pm, vals))
-      if vals and not isinstance(vals, list) and not isinstance(config[pm], vals):
+      if vals and not isinstance(vals,
```

Comment:
Can you change remove autoformatting for code style from this file, since it's not necessary and some decisions are visually not nice, like this if splitting in two lines? If it's just the log_summaries_from_dict functions that has changed a little bit, than it should be fine to just add those changes and don't change anything else, so that it will be easier to track changes if something brakes later.